### ISSUE TITLE
Document SASS and CSS support (or lack)

### ISSUE THREAD

**Issue by NateRadebaugh** [preconstruct/preconstruct#268]
I don't see anything here that defines SASS or CSS compilation support. If preconstruct doesn't support this itself, it'd be good to mention that, or document what workarounds may exist to output a `.css` file in `dist` folders.

**Comment by NateRadebaugh** [comment]
I can import the raw SCSS from a next app in the monorepo but it's not clear the recommended configuration for exporting CSS in the package, or how to best pick the files to publish with the packages.

**Comment by Andarist** [comment]
You are right - there is no CSS support right now. Preconstruct's main goal is to build JS packages which can be consumed by tools like node, bundlers, etc "as is". CSS always require custom setup because there is no standard way of shipping and consuming stylesheets to/from npm.

**Comment by mitchellhamilton** [comment]
Happy to accept a PR adding what @Andarist said to the docs.
